multi/imxrt-multi/uart.c

```diff
@@ -12,22 +12,18 @@
  */
 
 
-#include <string.h>
-#include <stdint.h>
-#include <stdio.h>
 #include <errno.h>
+#include <stdint.h>
```

Comment:
`string.h` is needed for `memcpy`:
![image](https://github.com/phoenix-rtos/phoenix-rtos-devices/assets/3939983/e48ec150-741c-45fd-bdca-360b71dcb0fc)